<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManagerForm</class>
 <widget class="QWidget" name="ClientManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,7,6,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="clientInfoLabel">
           <property name="font">
            <font>
             <family>휴먼매직체</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ClientInfoManager</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="clientFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="idLabel">
           <property name="text">
            <string>I&amp;D</string>
           </property>
           <property name="buddy">
            <cstring>idLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="idLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Auto-generated</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>&amp;Name</string>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="phoneNumberLabel">
           <property name="text">
            <string>&amp;Phone Number</string>
           </property>
           <property name="buddy">
            <cstring>phoneNumberLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="phoneNumberLineEdit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="placeholderText">
            <string>00(0)-000(0)-0000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>&amp;Address</string>
           </property>
           <property name="buddy">
            <cstring>addressLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="addressLineEdit"/>
         </item>
         <item row="4" column="0" colspan="2">
          <layout class="QHBoxLayout" name="addHorizontalLayout">
           <item>
            <widget class="QPushButton" name="addPushButton">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifyPushButton">
             <property name="text">
              <string>&amp;Modify</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>&amp;Search</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Name</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Phone Number</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Address</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="searchPushButton">
              <property name="text">
               <string>S&amp;earch</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="statePushButton">
              <property name="text">
               <string>All Info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>165</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="clientTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
